res/css/views/elements/_GenericEventListSummary.pcss

```diff
@@ -21,6 +21,16 @@ limitations under the License.
         margin-right: $spacing-8;
     }
 
+    .mx_GenericEventListSummary_spacer {
+        border-bottom: 1px solid $primary-hairline-color;
+        margin-left: 63px; /* TODO: Use a varilable */
+        line-height: $font-30px; /* TODO: Use a varilable */
```

Comment:
```suggestion
        margin-left: 63px; /* TODO: Use a variable */
        line-height: $font-30px; /* TODO: Use a variable */
```

Reply:
Thanks for finding them! I didn't notice at all..